Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/fragments/SelectorFragment.kt

```diff
@@ -36,6 +36,7 @@ import com.example.android.camera2.slowmo.R
 /**
  * In this [Fragment] we let users pick a camera, size and FPS to use for high
  * speed video recording
+ * Update: This part has been automated to select a lowest working setting for SLATE.
  */
 class SelectorFragment : Fragment() {
```

Comment:
You shouldn't need the fragment at all any more right?  Can you simplify this down and remove the fragment?

Reply:
I realised that there's a bug which is making the app crash on some chosen settings. This means that automating the selection of the last setting from the list will not work. I'm chasing that bug. This PR will be **blocked** till then.
Upon that I will simplify down this fragment.

**Ref:** [TODO](https://github.com/googleinterns/screen-latency-testing/blob/d934ba1b444d3076ff2c255146d4774fb8149d4f/Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/VideoProcessor.java#L67)